test-1.py

```diff
@@ -0,0 +1,69 @@
+
+from curses.ascii import isdigit
+import random
+
+min = 1
+max = 100
+number = random.randint(min, max)
+name = input("Wie lautet ihr Name: ")
+
+while len(name) == 0:
+    name = input("Bitte geben Sie erneut Ihren Namen ein: ")
+   
+print(f'Hallo, {name}! Ich denke an eine Zahl zwischen 1 und 100. Welche Zahl ist es?.')
```

Comment:
You can use the variables for min and max in this string instead of hard writing them.